src/commands/networks.ts

```diff
@@ -0,0 +1,61 @@
+import { providers } from "ethers";
+
+type SupportedNetwork = {
+  chainId: number;
+  explorer: string;
+  provider: providers.Provider;
+};
+
+export enum NetworkCmdName {
+  local = "local",
+  mainnet = "mainnet",
+  ropsten = "ropsten",
+  rinkeby = "rinkeby",
+  goerli = "goerli",
+  polygon = "polygon",
+  mumbai = "mumbai",
+}
+
+export const supportedNetwork: {
+  [key in NetworkCmdName]: SupportedNetwork;
+} = {
+  [NetworkCmdName.local]: {
+    chainId: 1337,
+    explorer: "https://localhost/explorer",
+    provider: new providers.JsonRpcProvider(),
+  },
```

Comment:
I was thinking, for future networks we want to add and support, we can just add them to this file.

Reply:
this really helps alot thanks 🙏🙏